lib/humidifier/version.rb

```diff
@@ -1,4 +1,4 @@
 module Humidifier
   # Gem version
-  VERSION = '1.0.6'.freeze
+  VERSION = '1.2.1'.freeze
```

Comment:
To stay in line with the AWS CloudFormation resource specification version (so people know which version is which)